docs/walkthrough/walkthrough_data.md

```diff
@@ -0,0 +1,92 @@
+  barcodes ([Other data](#other-data)) were added to the files for 
+  testing lossless conversion and de-multiplexing.
+
+For both unmapped and mapped reads, the sort order is _unsorted_ but 
+they are grouped by queryname (not reflected in the header). Mapped
+reads were generated by [bwa]({{site.data.software.bwa}}).
+
+## Reads data summary
+
+A brief description for each dataset is included in the following table.
+
+{% include note.html 
+content="Click on the file name to download the file." %}
+
+<!-- Note: Try to keep the columns to 3 or 4 columns only.
+If you add 5+ columns, your table may create horizontal scrolling with 
+the theme. Additionally, keep the column heading titles short. -->
+<table class="display">
+    <thead>
+        <tr class="header">
+            <th style="text-align:left" style="vertical-align:middle">File(s)</th>
+            <th style="text-align:center" style="vertical-align:middle">Format</th>
+            <th style="text-align:center" style="vertical-align:middle">Pair-end</th>
+            <th style="text-align:center" style="vertical-align:middle">Quality<br>encoding</th>
+            <th style="text-align:left" style="vertical-align:middle">Read<br>name</th>
+            <th style="text-align:left" style="vertical-align:middle">Indexing</th>
+            <th style="text-align:left" style="vertical-align:middle">Barcode<br>encoding</th>
+        </tr>
+    </thead>
+    <tbody>
+        {% for example_data in site.data.walkthrough_data %}
+        <tr>
+            <td style="text-align:left" style="vertical-align:middle">
+            {% for file in example_data.files %}
+                <a href="{{walkthrough_folder}}{{file}}">{{file}}</a>
+                {% unless forloop.last %}<br>{% endunless %}
+            {% endfor %}
+                
+            </td>
+            <td style="text-align:center" style="vertical-align:middle">{{example_data.format}}</td>
+            <td style="text-align:center" style="vertical-align:middle">{{example_data.paired}}</td>
+            <td style="text-align:center" style="vertical-align:middle">{{example_data.quality}}</td>
+            <td style="text-align:left" style="vertical-align:middle">{{example_data.readname}}</td>
+            <td style="text-align:left" style="vertical-align:middle">{{example_data.indexing}}</td>
+            <td style="text-align:left" style="vertical-align:middle">{{example_data.barcode}}</td>
+        </tr>
+        {% endfor %}
+    </tbody>
+
+</table>
+
+## Other data
+
+Additional data was included for its use in testing:
+
+* [2L.fragment.fa]({{walkthrough_folder}}2L.fragment.fa)/
+  [2L.fragment.fa.fai]({{walkthrough_folder}}2L.fragment.fa.fai): 
+  Indexed FASTA file for use with mapped reads and CRAM sources.
+* [dual.barcodes]({{walkthrough_folder}}dual.barcodes): barcode file 
+  for samples with dual-indexing.
+* [unique.barcodes]({{walkthrough_folder}}unique.barcodes): barcode 
```

Comment:
usually, you are referring to single barcoding. Here you talk about unique barcoding. Could maybe be a bit misleading...